scripts/rollup/build.js

```diff
@@ -468,6 +470,7 @@ function getPlugins(
             ? "'production'"
             : "'development'",
           __EXPERIMENTAL__,
+          __NEXT_MAJOR__,
```

Comment:
Instead of setting this up as a global I was suggesting that it would just be a constant inside ReactFeatureFlags.js so that it doesn't leak. Since it's a temporary state it's annoying to go around update tests and other things that might be flagged on this.

Reply:
Ok fixed